File: CodeForces/Java/CF_59A_Word.java
//  Problem: 59A. Helpful Maths
//  Link: https://codeforces.com/problemset/problem/59/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    String word = scan.next();
    int upper = 0;
    scan.close();
  
    for (char c : word.toCharArray()){
      if(c>='A' && c<='Z'){
        upper++;
      }
    }
    if(upper>word.length()/2){
      word = word.toUpperCase();
    } else{
      word = word.toLowerCase();
    }
    System.out.println(word);

  }
}

File: CodeForces/Python/CF_59A_Word.py
# Problem: 59A. Helpful Maths
# Link: https://codeforces.com/problemset/problem/59/A

word = input()
upper = 0
for i in word:
  if (i >= 'A' and i <= 'Z'):
    upper += 1

if (upper>len(word)/2):
  print(word.upper())
else:
  print(word.lower())